packages/core/src/scratchorg/pool/ClientSourceTracking.ts

```diff
@@ -60,31 +60,49 @@ export default class ClientSourceTracking {
 
             const sfpowerscriptsArtifacts = await this.org.getInstalledArtifacts();
 
+            //clean up MPD to just one package, so that source tracking lib
+            //does do a full scan and break
+            this.cleanupSFDXProjectJsonTonOnePackage(tempDir.name, sfpowerscriptsArtifacts[0].Name);
+
             const project = await SfdxProject.resolve(tempDir.name);
 
             // Create local source tracking files in temp repo
             const tracking = await SourceTracking.create({
                 org: this.org,
                 project: project,
             });
-            tracking.ensureLocalTracking();
```

Comment:
Why is the initialization for source tracking removed?

Reply:
Not required, as the updateLocalTracking calls it anyway